core/model/scores.go

```diff
@@ -40,7 +40,8 @@ type Score struct {
 	AppID                  string       `json:"app_id" bson:"app_id"`
 	UserID                 string       `json:"user_id" bson:"user_id"`
 	SurveyType             string       `json:"survey_type" bson:"survey_type"`
-	ExternalProfileID      string       `json:"external_profile_id" bson:"external_profile_id"`
+	ExternalProfileID      string       `json:"external_profile_id" bson:"external_profile_id"`     // Mastodon ID (legacy)
+	ExternalUserID         string       `json:"external_user_id,omitempty" bson:"external_user_id"` // AmgUUID (new)
```

Comment:
I think we need to update the docs to reflect that this new `external_user_id` field   is available. We should also mark the `external_profile_id` as deprecated.